src/test/Makefile

```diff
@@ -131,59 +131,53 @@ distclean: clean
 # Rules
 ##################################################################
 
-ifeq ($(CUCC), )
-   OBJ_SUFFIX=o
-else
-   OBJ_SUFFIX=obj
-endif
-
 # C
 
-ij: ij.${OBJ_SUFFIX}
+ij: ij.o
 	@echo  "Building" $@ "... "
 	${LINK_CC} -o $@ $< ${LFLAGS}
 
-ij_assembly: ij_assembly.${OBJ_SUFFIX}
+ij_assembly: ij_assembly.o
 	@echo  "Building" $@ "... "
 	${LINK_CC} -o $@ $< ${LFLAGS}
 
 sstruct: sstruct.o
 	@echo  "Building" $@ "... "
 	${LINK_CC} -o $@ $< ${LFLAGS}
 
-struct: struct.${OBJ_SUFFIX}
+struct: struct.obj
```

Comment:
I'm not following what's going on here.  Why are some of the drivers using `.o` extensions and others `.obj`?